Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -65,7 +66,7 @@
                 .withStorageKey(this.storageKey)
                 .withFixedSourceUrl(
                     `/odata/Organizations(${user.currentOrganizationId
-                    })/ItSystemUsageOverviewReadModels?$expand=RoleAssignments`)
+                    })/ItSystemUsageOverviewReadModels?$expand=RoleAssignments,DataProcessingRegistrations`)
```

Comment:
Hvad med expand på de andre felter som ItProjects? Hvis de ikke er nødvendige hvorfor er DataProcessingRegistrations så nødvendig?

Reply:
Man skal kun expande det som man skal have med tilbage. Man behøves ikke for at søge og der er ingen grund til at expande hvis det er nok at rendere csv :-)